pkg/tablebackup/tablebackup.go

```diff
@@ -27,10 +27,10 @@ import (
 
 const (
 	// TODO: move those constants somewhere else
-	DeltasDirName              = "deltas"
-	BasebackupFilename         = "basebackup.copy"
-	BaseBackupStateFileName    = "info.yaml"
-	BackupStateFileNamePattern = "backup_state_%d.yaml"
```

Comment:
unfortunately I missed that in the previous PR. there are no tableOID-suffixed files in the table folder, so why to introduce.

Reply:
I put a comment about this below. The idea is to be redundant in order to help us in case the file accidentally ends up in the wrong place (and there seems to be no downsides)